crypto/dso/dso_dlfcn.c

```diff
@@ -307,6 +307,71 @@ static int dladdr(void *address, Dl_info *dl)
     return (int)v;
 }
 # endif                         /* __sgi */
+# ifdef _AIX
```

Comment:
It's surely not very sexy question, but nevertheless, it's whether or not loadquery was available all along or introduced recently. What I'm hinting at is that this macro might be _AIX53 or something. Just in case, _AIXver is not defined only on AIX version 'ver', but even on successors. So that it's defined on 5.3, 6.1, 7.1, etc.

Reply:
FWLIW, my 5.3 system has headers for loadquery that match AIX 7.1 headers (not able to build/test directly at the moment)

Reply:
Well, one can argue that 5.3 is old enough to claim that it was around all along...

Reply:
Well, I am unable to dig out any hard evidence as to when loadquery() was actually introduced. But I found a red book claiming to be from Dec. 1999, called "AIX 4.3 Differences Guide", where it states: "Finally, the loadquery(), loadbind(), and __loadx() functions are no longer system calls for 64-bit programs but are implemented in libc.a." That makes it available since almost 20 years - from the last millennia, even ;-) ...
Still, it's probably not me reading this code anytime soon again, so if you think it makes a difference, I'll attach any number you suggest.

Reply:
OK, let me rephrase, *I* would argue that 5.3 is old enough to claim it was around all along. And even more so since there is evidence that it available even earlier. In other words _AIX is fine.